best_practices/security.rst

```diff
@@ -167,7 +166,7 @@ need to repeat the expression code using Twig syntax:
         <a href=""> ... </a>
     {% endif %}
 
-The easiest solution - if your logic is simple enough - is to add a new method
+A better solution - if your logic is simple enough - is to add a new method
```

Comment:
Easy !== better. What about "Another solution" ?

Reply:
What about `A good solution can be to...`